src/foam/u2/crunch/CapabilityRequirementView.js

```diff
@@ -110,7 +110,12 @@ foam.CLASS({
 
         .start().addClass('titles')
           // title
-          .start().addClasses([this.myClass('title'), 'h400']).translate(mainCapability[0].id+'.name', mainCapability[0].name).end()
+          .start().addClasses([this.myClass('title'), 'h400'])
+            .callIfElse(mainCapability[0].marketingHeader,
+              function() { this.translate(mainCapability[0].id+'.marketingHeader', mainCapability[0].marketingHeader); },
+              function() { this.translate(mainCapability[0].id+'.name', mainCapability[0].name); }
```

Comment:
If the default is the name - do you need an ifElse here?

Reply:
@Nauna 
i did something like: 
```.start().addClasses([this.myClass('title'), 'h400']).translate(mainCapability[0].id+'.marketingHeader', mainCapability[0].name).end()```,
but if we want to display marketingHeader without translation we don't have this option.
Should this option be considered?